modules/tinymce/src/themes/silver/test/ts/browser/editor/DefaultHeightTest.ts

```diff
@@ -0,0 +1,52 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { McEditor } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.DefaultHeightTest', () => {
+  const baseSettings = {
+    base_url: '/project/tinymce/js/tinymce'
+  };
+
+  it('TINY-8443: Default height should be 400px', async () => {
+    const editor = await McEditor.pFromSettings<Editor>(baseSettings);
+    assert.equal(editor.getContainer().offsetHeight, 400);
+    McEditor.remove(editor);
+  });
+
+  it('TINY-8443: Option heigh takes precedence', async () => {
+    const editor = await McEditor.pFromSettings<Editor>({ height: 500, ...baseSettings });
+    assert.equal(editor.getContainer().offsetHeight, 500);
+    McEditor.remove(editor);
+  });
+
+  it('TINY-8443: CSS height takes precendence in the absence of option height', async () => {
+    let editor = await McEditor.pFromHtml<Editor>('<div style="height: 600px"></div>', baseSettings);
+    assert.equal(editor.getContainer().offsetHeight, 600);
```

Comment:
This assertion is repeated a lot here... can we pull it out into a helper function? e.g.
```
const assertEditorHeight = (editor: Editor, expected: number) =>
  assert.equal(editor.getContainer().offsetHeight, expected);
```